install.sh

```
@@ -1,170 +1,371 @@
+if [[ -n "$script_dir" && -f "$script_dir/package.json" ]]; then
+  local_source=1
+  if [[ ! -f "$script_dir/dist/wt.cjs" ]]; then
+    echo "wt installer: dist/wt.cjs is missing; run npm ci && npm run build first." >&2
     exit 1
   fi
+fi
+
+platform="$(uname -s)"
+[[ "$platform" == Darwin || "$platform" == Linux ]] || fail "standalone installation supports macOS and Linux only. Use npm install -g @absolutepraya/wt on Windows."
+if command -v curl >/dev/null 2>&1; then downloader=curl
+elif command -v wget >/dev/null 2>&1; then downloader=wget
```

Shellcheck (0.11.0)

[warning] downloader appears unused. Verify use (or export if used externally).

(SC2034)